apps/aecore/lib/aecore/tx/transaction.ex

```diff
@@ -18,10 +19,14 @@ defmodule Aecore.Tx.Transaction do
 
       @spec gas_price :: non_neg_integer()
       def gas_price do
-        GovernanceConstants.tx_base_gas(__MODULE__)
+        if Enum.member?(GovernanceConstants.get_valid_txs_type(), __MODULE__) do
+          GovernanceConstants.default_tx_gas_price()
+        else
+          GovernanceConstants.block_gas_limit()
```

Comment:
my old comment regarding this still applies

Reply:
Could you clarify your thoughts on this? I don't really get it

Reply:
"returning the limit as gas price for each transaction doesn't seem to make sense"

Why is the `block_gas_limit` returned if we don't know this transaction type?

Reply:
This is how Epoch node [handles it](https://github.com/aeternity/epoch/blob/v0.25.0/apps/aecore/src/aec_governance.erl#L121). They allow "unknown" transactions,  but the gas price for it is, by default: `block gas limit`. They might handle it somewhere , but I don't think that this in the scope of this issue.

Reply:
Okay, thanks for clarifying

Reply:
How would you get an unknown transaction unless you write the code yourself? Is this really needed?